ext/crypto/Cargo.toml

```diff
@@ -19,11 +19,12 @@ base64 = "0.13.0"
 block-modes = "0.8.1"
 deno_core = { version = "0.110.0", path = "../../core" }
 deno_web = { version = "0.59.0", path = "../web" }
-elliptic-curve = "0.10.6"
+elliptic-curve = "0.11.5"
 lazy_static = "1.4.0"
 num-traits = "0.2.14"
-p256 = { version = "0.9.0", features = ["ecdh"] }
-p384 = "0.8.0"
+p256 = { version = "0.10.0", features = ["ecdh", "pem"] }
```

Comment:
What do you need the `pem` feature for?

Reply:
Don't need PEM. It was the only way I could find to satisfy constraint for trait `FromPrivateKey` to use `from_pkcs8_der` on `p256::SecretKey<NistP256>`. Line ~730 import_key.rs

Reply:
Yeah, have unfortunately discovered this myself now during testing.